### PR TITLE
Pia 4048 improve gini bank sdk example app

### PR DESCRIPTION
**Pull request by ValentinaIancu-Gini**

### CODE REVIEW

**Comment by ValentinaIancu-Gini** [comment]
@zladzeyka I've tested on multiple devices and nothing weird was found. I will copy-paste your previous findings here:
```
1. when we have disabled the return reasons dialog UI is blocked on the Digital Invoice overview screen - I can't deselect the line items
2. Onboarding is always shown + multipage is enabled. Each time when we use Add pages the Review screen to Camera -> Onboarding is shown again 🤨, we need to check it. Also, I found that this is happening also when having enabled the custom bottom navigator bar and onboarding at every launch -> launch the capture and tap on the help button and tap back to go to the camera screen the onboarding screen is appearing once again

```

File: BankSDK/GiniBankSDKExample/GiniBankSDKExample/Info.plist
Comment (zladzeyka) [inline BankSDK/GiniBankSDKExample/GiniBankSDKExample/Info.plist:1]: This change looks suspicious - I think it came from Xcode 15 beta

**Review by zladzeyka — APPROVED** [review]
Great job!

File: BankSDK/GiniBankSDKExample/GiniBankSDKExample/Info.plist
Comment (ValentinaIancu-Gini) [reply BankSDK/GiniBankSDKExample/GiniBankSDKExample/Info.plist:1]: :-o maybe... I will take a look and see if I can fix this.

Comment (ValentinaIancu-Gini) [reply BankSDK/GiniBankSDKExample/GiniBankSDKExample/Info.plist:1]: @zladzeyka I fixed it.

**Review by zladzeyka — APPROVED** [review]

**Pull request merged**